components/list-issues.tsx

```diff
@@ -164,15 +166,13 @@ export default function ListIssues({
   }
 
   function handlerSearch() {
-    if (!appState.Service?.network?.active?.name) return;
+    if (router.pathname === "/[network]" && !networkName ||
```

Comment:
switch it around, since `networkName` is used on both conditions, we can shorten this:

```ts
if (!networkName && (router.pathname === "/[network]" || router.pathname.includes('...) && !chain)
```